### PR TITLE
Fabo/lunie extension

### LINKED ISSUE #217: Add Lunie extension support
To accomodate late migration issues, we should allow Lunie extension support in Lunie3.

### PR DESCRIPTION
**Pull request by faboweb**
closes #217

client side is working
need to test signing and return values

needs https://github.com/luniehq/lunie/pull/5214

### CODE REVIEW

**Comment by Bitcoinera** [comment]
image

:eyes: ? It says Keplr in here, not Lunie ?

**Comment by Bitcoinera** [comment]
>  ? It says Keplr in here, not Lunie ?

OK, I see. There is a logic problem here, I find. It should display in this case, "You have no accounts on the extension. Create an account first to start using Lunie"

**Comment by Bitcoinera** [comment]
I think we can improve the styles here:

image

File: pages/extension/index.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
       <div class="session-main">
         <Button
-          value="Connect Keplr"
+          value="Connect Lunie Extension"
```
Comment (jbibla) [inline pages/extension/index.vue:8]: doesn't this need to work for both Keplr and Lunie extensions?

Comment (Bitcoinera) [reply pages/extension/index.vue:8]: Keplr has another section in CardSignInRequired

File: components/common/Address.vue [hunk 2]
```diff
@@ -17,7 +17,7 @@
 import { formatAddress } from '~/common/address'
 
 export default {
-  name: `Address`,
+  name: `LunieAddress`,
```
Comment (jbibla) [inline components/common/Address.vue:20]: this can be reverted i think

File: pages/extension/index.vue (hunk 1 above)
Comment (jbibla) [reply pages/extension/index.vue:8]: CardSignInRequired isn't a component anymore. but ok, i see.

File: pages/keplr/index.vue [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="session-container">
+    <h2 class="session-title">Use Keplr Browser Extension</h2>
+
+    <div v-if="error" class="error-container">
+      <p>There was an error connecting to the Keplr extension:<br /></p>
+      <p class="error">{{ error }}</p>
+    </div>
+
+    <div v-else-if="loading" class="session-main">
```
Comment (jbibla) [inline pages/keplr/index.vue:10]: this should use the same loading logic and page structure as the lunie extension...

File: signing/signer.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ export async function getSigner(signingType, { address, password }, chainId) {
     return wallet
   } else if (signingType === `ledger`) {
     return await getLedger()
-  } else if (signingType === `extension`) {
+  } else if (signingType === `keplr`) {
```
Comment (jbibla) [inline signing/signer.js:15]: this is going to work for lunie ext and keplr?

**Comment by Bitcoinera** [comment]
Tried sending with extension and it worked

File: components/common/Address.vue (hunk 2 above)
Comment (faboweb) [reply components/common/Address.vue:20]:
```suggestion
  name: `Address`,
```

File: signing/signer.js (hunk 4 above)
Comment (faboweb) [reply signing/signer.js:15]: no, lunie extension is using a different logic

**Pull request merged**